### PR TITLE
Cgmes update simple update

### PR DESCRIPTION
**Pull request by kaltakovae**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
New feature

**What is the new behavior (if this is a feature change)?**
This feature implements update functionality for CGMES data with changes made in IIDM model. The following is included:
-  Feature for copy/update of CGMES model (based on underlying copy/update in triplestores).
-  Feature for building a changelog of IIDM changes, stored in CGMES model extension.
-  Support for variants in changelogs.
-  Feature for converting a changelog of IIDM changes into a sequence of triplestore update commands.
- SPARQL update query.
-  Feature for exporting updated CGMES model.

Current implementation supports simple updates (IIDM attributes that can be mapped to CGMES attributes in main objects). Further iterations will include more complex object modifications. Currently supported changes are:

- Generator targetP, targetQ, maxP, minP, ratedS;
- Load p0, q0;
- Shunt bPerSection, maximumSectionCount, currentSectionCount;
- Line r, x, g1, b1, g2, b2 (CGMES has only bch, gch).

To verify the update function, the test unit implemented does the following:

- Import CGMES individual grid model.
- Perform load/generation scaling. Changes in all (p, q) values of loads and generators
- Run Load Flow completion results.
- Export updated CGMES.
- Import the exported CGMES and compare resulting IIDM Network with initial one.

### CODE REVIEW

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java [hunk 1]
```diff
@@ -101,12 +109,23 @@ public String getFormat() {
 
     @Override
     public Network importData(ReadOnlyDataSource ds, NetworkFactory networkFactory, Properties p) {
+        if (profiling != null) {
+            profiling.start();
+        }
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java]: You should use the same design: a lazy implementation or check in CgmesExport that the profiling is not null. Try to be consistent in both classes

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesModelExtension.java [hunk 2]
```diff
@@ -18,15 +19,21 @@
...
+        this.cgmesUpdate = Objects.requireNonNull(cgmesUpdate);
     }
 
     public CgmesModel getCgmesModel() {
         return cgmes;
     }
 
+    public CgmesUpdate getCgmesUpdate() {
+        return cgmesUpdate;
+    }
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesModelExtension.java]: Checked: it seems CgmesUpdate is never null, you can remove assertions in CgmesExport.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/CgmesUpdate.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        if (!(change instanceof IidmChangeUpdate)) {
+            throw new ConversionException("Unsupported conversion to CGMES for IIDM change type " + change.getClass().getSimpleName());
+        }
+    }
+
+    private IidmToCgmes findConversion(IidmChange change, UpdateContext context) {
+        if (context.cimVersion == 16) {
+            return IIDM_TO_CGMES16.findConversion(change);
+        }
+        return null;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/CgmesUpdate.java]: Return null or throw UnsupportedFormatException?

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/CgmesUpdate.java [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+            return actualContexts;
+        }
+
+        private final String basename;
+        private final Map<String, String> actualContexts = new HashMap<>();
+        private final Profiling profiling;
+        private final int cimVersion;
+    }
+
+    private Changelog changelog;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/CgmesUpdate.java]: Should be final?

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+ */
+public class Changelog implements NetworkListener {
+
+    /**
+     * Register Network changes
+     *
+     * @param network the IIDM Network
+     */
+    public Changelog(Network network) {
+        network.addListener(this);
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java]: Assert that network is not null

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+            changesByVariant.computeIfAbsent(variantId, k -> new ArrayList<>()).add(new IidmChangeUpdate(identifiable, attribute, oldValue, newValue));
+        }
+    }
+
+    @Override
+    public void onVariantCreated(String sourceVariantId, String targetVariantId) {
+        // Will overwrite any previous changelog saved for target variant
+        List<IidmChange> sourceChanges = changesByVariant.get(sourceVariantId);
+        if (sourceChanges != null) {
+            changesByVariant.put(targetVariantId, new ArrayList<>(sourceChanges));
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java]: Why do you wrap (copy) the sourceChanges to a new list?

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+        } else {
+            List<IidmChange> cs = new ArrayList<>(baseChanges.size() + changesByVariant.size());
+            cs.addAll(baseChanges);
+            cs.addAll(changesByVariant.get(variantId));
+            return Collections.unmodifiableList(cs);
+        }
+    }
+
+    private List<IidmChange> baseChanges;
+    private Map<String, List<IidmChange>> changesByVariant;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java]: Should be declared final

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmChangeUpdate.java [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+ * @author Elena Kaltakova <kaltakovae at aia.es>
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class IidmChangeUpdate extends IidmChange {
+
+    public IidmChangeUpdate(Identifiable identifiable, String attribute, Object oldValue, Object newValue) {
+        super(identifiable);
+        this.attribute = Objects.requireNonNull(attribute);
+        this.oldValue = oldValue;
+        this.newValue = Objects.requireNonNull(newValue);
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmChangeUpdate.java]: Are you sure that newValue is never null? As oldValue can be null, I guess newValue could also

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmToCgmes.java [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+ * @author Elena Kaltakova <kaltakovae at aia.es>
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class IidmToCgmes {
+
+    public List<TripleStoreChange> convert(IidmChange change) {
+        if (change instanceof IidmChangeUpdate) {
+            return convertUpdate((IidmChangeUpdate) change);
+        } else {
+            return Collections.emptyList();
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmToCgmes.java]: Is it a programming issue if change is not a IidmChangeUpdate object? Should we throw an exception or at least warn?

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+        cgmes.add(CgmesSubset.STATE_VARIABLES, "SvVoltage", voltages);
+
+        PropertyBags powerFlows = new PropertyBags();
+        for (Load l : n.getLoads()) {
+            PropertyBag p = createPowerFlowProperties(cgmes, l.getTerminal());
+            if (p != null) {
+                powerFlows.add(p);
+            } else {
+                // FIXME CGMES SvInjection objects created as loads
+                System.err.println("No SvPowerFlow created for load " + l.getId());
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java]: Don't use System.err: use a logger instead.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+        }
+        PropertyBag p = new PropertyBag(SV_POWERFLOW_PROPERTIES);
+        p.put("p", fs(terminal.getP()));
+        p.put("q", fs(terminal.getQ()));
+        p.put(CgmesNames.TERMINAL, cgmesTerminal);
+        return p;
+    }
+
+    private static String fs(double value) {
+        return "" + value;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java]: Use Double.toString() ?

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+    }
+
+    private static String is(int value) {
+        return "" + value;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java]: Use Integer.toString?

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreChange.java [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+/**
+ * @author Elena Kaltakova <kaltakovae at aia.es>
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class TripleStoreChange {
+
+    public TripleStoreChange(String queryName, String subject, TripleStoreChangeParams params) {
+        this.queryName = queryName;
+        this.subject = subject;
+        this.params = params;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreChange.java]: Assert that everything is not null?

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreChangeParams.java [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+ * @author Elena Kaltakova <kaltakovae at aia.es>
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class TripleStoreChangeParams {
+
+    public TripleStoreChangeParams(TripleStoreSimpleUpdateReference ref, String value) {
+        this.predicate = ref.predicate();
+        this.contextReference = ref.contextReference();
+        this.valueIsUri = ref.valueIsUri();
+        this.value = value;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreChangeParams.java]: Assert that ref and value are not null?

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreComputedValueUpdateReference.java [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+import com.powsybl.iidm.network.Identifiable;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class TripleStoreComputedValueUpdateReference extends TripleStoreSimpleUpdateReference {
+
+    public TripleStoreComputedValueUpdateReference(String predicate, String contextReference, Function<Identifiable, String> valueComputation) {
+        super(predicate, contextReference);
+        this.valueComputation = valueComputation;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreComputedValueUpdateReference.java]: Assert that valueComputation are not null?

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreSimpleUpdateReference.java [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+public class TripleStoreSimpleUpdateReference {
+
+    public TripleStoreSimpleUpdateReference(String predicate, String contextReference) {
+        this(predicate, contextReference, false);
+    }
+
+    public TripleStoreSimpleUpdateReference(String predicate, String contextReference, boolean valueIsUri) {
+        this.predicate = predicate;
+        this.contextReference = contextReference;
+        this.valueIsUri = valueIsUri;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreSimpleUpdateReference.java]: Assert everything is not null

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/IidmToCgmes16.java [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+        } else if (o instanceof Line) {
+            return line;
+        } else if (o instanceof TwoWindingsTransformer) {
+            return t2;
+        } else if (o instanceof ShuntCompensator) {
+            return shunt;
+        } else if (o instanceof VoltageLevel) {
+            return vl;
+        } else {
+            return null;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/IidmToCgmes16.java]: Add a debug info to display the type of `o` that is not (yet) managed.

File: cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/update/NetworkChanges.java [hunk 18]
```diff
@@ -0,0 +1,169 @@
...
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public final class NetworkChanges {
+    private NetworkChanges() {
+    }
+
+    public static void scaleLoadGenerator(Network network, int maxChanges) {
+        int count;
+        count = 0;
```
Comment (mathbagu) [inline cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/update/NetworkChanges.java]: int count = 0;

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java (hunk 6 above)
Comment (zamarrenolm) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java]: Specific changes by variant are maintained in separate lists. When new variant is created the changelog for the target variant must contain all the changes made in the source up to this point. If additional changes are performed in the source variant after the new variant is created, they must not be present in the target variant changelog, so a copy is required. New changes on the target variant must be added only to the changelog of the target variant.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmToCgmes.java (hunk 9 above)
Comment (zamarrenolm) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmToCgmes.java]: You are right, for the moment these are changes that are not supported. Should thrown an UnsupportedOperationException.

File: cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/update/NetworkChanges.java (hunk 18 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/update/NetworkChanges.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/IidmToCgmes16.java (hunk 17 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/IidmToCgmes16.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreSimpleUpdateReference.java (hunk 16 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreSimpleUpdateReference.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreComputedValueUpdateReference.java (hunk 15 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreComputedValueUpdateReference.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreChangeParams.java (hunk 14 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreChangeParams.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreChange.java (hunk 13 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/TripleStoreChange.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java (hunk 12 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java (hunk 11 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java (hunk 10 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmToCgmes.java (hunk 9 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmToCgmes.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmChangeUpdate.java (hunk 8 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmChangeUpdate.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java (hunk 7 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java (hunk 5 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/CgmesUpdate.java (hunk 4 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/CgmesUpdate.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/CgmesUpdate.java (hunk 3 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/CgmesUpdate.java]: Done

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesModelExtension.java (hunk 2 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesModelExtension.java]: ok

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java (hunk 1 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java]: As Luma commented, we will remove profiling from the code.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java [hunk 19]
```diff
@@ -64,87 +84,10 @@ public String getFormat() {
...
+        ADD_STATE_VARIABLES, WRITE_UPDATED_CGMES, CGMES_READ, CGMES_CONVERSION;
     }
 
-    private static final List<String> SV_VOLTAGE_PROPERTIES = Arrays.asList(CgmesNames.ANGLE, CgmesNames.VOLTAGE, "TopologicalNode");
-    private static final List<String> SV_POWERFLOW_PROPERTIES = Arrays.asList("p", "q", CgmesNames.TERMINAL);
-    private static final List<String> SV_SHUNTCOMPENSATORSECTIONS_PROPERTIES = Arrays.asList("ShuntCompensator",
-            "continuousSections");
-    private static final List<String> SV_TAPSTEP_PROPERTIES = Arrays.asList(CgmesNames.POSITION,
-            CgmesNames.TAP_CHANGER);
+    private Profiling profiling;
```
[the remark this answers is not in the document]
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java]: As agreed, we will remove profiling from the code.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java [hunk 20]
```diff
@@ -7,51 +7,71 @@
...
-        CgmesModel cgmes = ext.getCgmesModel();
+        CgmesUpdate cgmesUpdate = ext.getCgmesUpdate();
+        if (cgmesUpdate == null) {
+            throw new CgmesModelException("No update information for CGMES model");
+        }
 
-        // Clear the previous SV data
-        cgmes.clear(CgmesSubset.STATE_VARIABLES);
+        CgmesModel cgmesSource = ext.getCgmesModel();
+        profiling.start();
```
[the remark this answers is not in the document]
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java]: As agreed, we will remove profiling from the code.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java [hunk 21]
```diff
@@ -7,51 +7,71 @@
...
+        // without the need for the original CgmesModel
         CgmesModelExtension ext = network.getExtension(CgmesModelExtension.class);
         if (ext == null) {
             throw new CgmesModelException("No extension for CGMES model found in Network");
         }
-        CgmesModel cgmes = ext.getCgmesModel();
+        CgmesUpdate cgmesUpdate = ext.getCgmesUpdate();
+        if (cgmesUpdate == null) {
+            throw new CgmesModelException("No update information for CGMES model");
+        }
```
[the remark this answers is not in the document]
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java]: CgmesUpdate is asserted for notNull in CgmesModelExtension class. 
So, as you mentioned below, the assertion in CgmesExport can be removed.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java (hunk 20 above)
[the remark this answers is not in the document]
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java]: Profiling code is cleaned out

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java (hunk 1 above)
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java]: Profiling code is cleaned out

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java (hunk 19 above)
[the remark this answers is not in the document]
Comment (kaltakovae) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java]: Profiling code is cleaned out

**Pull request merged**